build.csx

```diff
@@ -0,0 +1,66 @@
+#r "System.IO.Compression.FileSystem.dll"
+#load "scripts/cmd.csx"
+#load "scripts/download.csx"
+#load "scripts/inspect.csx"
+#load "src/packages/simple-targets-csx.5.0.0/simple-targets.csx"
+
+using System;
+using System.IO;
+using System.IO.Compression;
+using static SimpleTargets;
+
+// locations
+var resharperCltUrl = "https://download.jetbrains.com/resharper/JetBrains.ReSharper.CommandLineTools.2016.2.20160913.100041.zip";
+var resharperCltPath = $"{Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData)}/.resharper/JetBrains.ReSharper.CommandLineTools.2016.2.20160913.100041.zip";
+var inspectCodePath = "./.resharper/inspectcode.exe";
+var msBuild = $"{Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86)}/MSBuild/14.0/Bin/msbuild.exe";
+var solution = "./src/NServiceBus.RabbitMQ.sln";
+var dotSettings = "./src/NServiceBus.RabbitMQ.sln.DotSettings";
+var resetBroker = "./scripts/Reset-RabbitMQ.ps1";
+var nunit = "./src/packages/NUnit.ConsoleRunner.3.4.1/tools/nunit3-console.exe";
+var unitTests = "./src/NServiceBus.RabbitMQ.Tests/bin/Release/NServiceBus.Transports.RabbitMQ.Tests.dll";
+var acceptanceTests = "./src/NServiceBus.RabbitMQ.AcceptanceTests/bin/Release/NServiceBus.RabbitMQ.AcceptanceTests.dll";
+var transportTests = "./src/NServiceBus.Transport.RabbitMQ.TransportTests/bin/Release/NServiceBus.Transport.RabbitMQ.TransportTests.dll";
+
+// targets
+var targets = new TargetDictionary();
+
+targets.Add("default", DependsOn("build", "inspect", "unit-test", "acceptance-test", "transport-test"));
+
+targets.Add("build", () => Cmd(msBuild, $"{solution} /p:Configuration=Release /nologo /m /v:m /nr:false"));
+
+targets.Add(
+    "download-resharper-clt",
+    () => { if (!File.Exists(resharperCltPath)) Download(resharperCltUrl, resharperCltPath); });
+
+targets.Add(
+    "unzip-resharper-clt",
+    DependsOn("download-resharper-clt"),
+    () => { if (!File.Exists(inspectCodePath)) ZipFile.ExtractToDirectory(resharperCltPath, Path.GetDirectoryName(inspectCodePath)); });
+
+targets.Add(
+    "run-inspectcode",
+    DependsOn("build", "unzip-resharper-clt"),
```

Comment:
Could this dependency chain be based on the existence of inspectcode.exe in such a way that if it's missing it would download on demand and unzip, vs having to keep the zip around all the time now?